services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/RouteOptions.java

```diff
@@ -2019,6 +2032,23 @@ public Builder snappingIncludeClosuresList(@Nullable List<Boolean> snappingClosu
     @NonNull
     public abstract Builder experimental(@Nullable Experimental experimental);
 
+    /**
+     * Use this method to add request parameters which are not present in model yet but supported
+     * on the Directions API, for example experimental parameters.
+     * @param unrecognizedProperties parameters to add to request
+     */
```

Comment:
Please highlight that this only work for URLs generated with `toUrl`. This doesn't work if someone uses `RouteOptions`  with `MapboxDirections` class for example.

Reply:
done